llvm/lib/Target/RISCV/RISCVInstrInfoCOREV.td

```diff
@@ -69,6 +69,19 @@ def cv_uimm12 : Operand<XLenVT>,
   let OperandNamespace = "RISCVOp";
 }
 
+def cv_tsimm6 : Operand<XLenVT>, TImmLeaf<XLenVT, [{return isInt<6>(Imm);}]> {
```

Comment:
The difference is that `cv_tsimm6` is a `TIMMLeaf` while `simm6` is an `IMMLeaf`, which corresponds to `TargetConstantSDNode` instead of `ConstantSDNode`